scene/main/viewport.h

```diff
@@ -228,7 +228,7 @@ class Viewport : public Node {
 	Transform2D global_canvas_transform;
 	Transform2D stretch_transform;
 
-	Size2i size = Size2i(512, 512);
+	Size2i size = DisplayServer::get_singleton()->window_get_size();
```

Comment:
In my opinion, this change is not necessary here.
Besides, analyzers complain about it.

Reply:
ohk, cool